cleanSweStore.pl

```diff
@@ -170,7 +176,19 @@ =head1 OPTIONS
 		print $LEFTONSWESTORE "$runfolder\t$key\n";		
 	}
 
-					
+	if($removeUnaligned) {
+		#Check if Unaligned folder exists
```

Comment:
better to save the path to a variable and reuse it in the statements below, e.g. `my $unalignedPath = $swestorePath/20$year-$month/$runfolder/Unaligned`